perses/tests/test_geometry_engine.py

```diff
@@ -1122,6 +1122,7 @@ def _guessFileFormat(file, filename):
     file.seek(0)
     return 'pdb'
 
+# TODO: This function is not being used by anything. Remove?
```

Comment:
Remove it!

Reply:
Is there a similar test somewhere else that does the same thing? 

I would be careful about removing stale code like this -- there are a lot of tests like this in perses (e.g. all of the code in `test_samplers.py` where the test is broken and/or not being run anywhere, but we do want to make sure there is test coverage for that particular feature and so at some point, we'll need to fix the tests. Naybe this is a larger topic to discuss at the next perses dev sync, but for now I would probably just leave the test in there.